Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -910,6 +943,7 @@ struct HFTrackIndexSkimsCreator {
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   return WorkflowSpec{
+    adaptAnalysisTask<SelectCollisions>(cfgc, TaskName{"hf-produce-sel-collision"}),
```

Comment:
It would be better to just call the class `HfProduceSelCollisions` and avoid `TaskName` argument.

Reply:
Ok! Should we do it also for the other two classes?

Reply:
Yes, please!

Reply:
Done! 👍